src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -875,8 +840,17 @@ def factoryWorkloadConstruction(self, workloadName, arguments):
 
         Named this way so that nobody else will try to use this name.
         """
-        self.masterValidation(schema=arguments)
-        self.validateSchema(schema=arguments)
+        # TODO: eventually perform a complete argument validation for Resubmission workflows
```

Comment:
these comments can be removed now, since we do have a spec validation for Resubmission in the spec `validateSchema`